components/dropdown/dropdown-menu.js

```diff
@@ -65,7 +81,15 @@ class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 		if (e.target.getAttribute('role') !== 'menuitemradio') {
 			return;
 		}
-		this.close();
+
+		// Add a small delay before closing the menu
+		setTimeout(() => {
+			this.closeRadio = true;
+			this.close();
+		}, closeTimeoutDuration);
+
+		// Wait until animation is complete before toggling value
+		setTimeout(() => { this.closeRadio = false; }, 100);
```

Comment:
We likely just want `this.closeRadio = false` in `_onClose` (called after the `d2l-dropdown-close` event) in this file, though you likely would want to test that that works out okay timing-wise to allow the animation to run.

Reply:
I moved it to _onClose but I still need to wait for the animation to complete before toggling it off. If I have it without the timeout the animation doesn't play at all or gets cut off